apps/projects/app/components/Panel/ReviewWork/ReviewWork.js

```diff
@@ -1,20 +1,224 @@
+            <SafeLink
+              href={submittor.url}
+              target="_blank"
+              style={{ textDecoration: 'none', color: '#21AAE7', marginRight: '6px' }}
+            >
+              {submittor.name ? submittor.name : submittor.login}
+            </SafeLink>
+            applied {submissionDateDistance} ago
+          </UserLink>
 
-const ReviewWork = ({ issues, rate, onSubmit }) => <div>Review Work</div>
-
-ReviewWork.propTypes = {
-  /** array of issues to allocate bounties on */
-  issues: PropTypes.arrayOf(
-    PropTypes.shape({
-      id,
-      level,
-    })
-  ),
-  /** base rate in pennies */
-  rate: PropTypes.number,
-  /** callback fn */
-  onSubmit: PropTypes.func,
+          <Separator/>
+
+          <FieldTitle>Proof of Work</FieldTitle>
+          <SafeLink
+            href={work.proof}
+            target="_blank"
+            style={{ textDecoration: 'none', color: '#21AAE7' }}
+          >
+            <DetailText>{work.proof}</DetailText>
+          </SafeLink>
+
+          <CommentsCheck comments={work.comments} />
+          <DetailText>{work.comments}</DetailText>
+
+          <FieldTitle>Hours Worked</FieldTitle>
+          <DetailText>{work.hours}</DetailText>
+
+        </SubmissionDetails>
+
+        <FormField 
+          label="Quality Rating" 
+          required
+          input={
+            <DropDown
+              items={ratings}
+              onChange={this.generateRatingChange()}
+              active={this.state.rating}
+            />
+          }
+        />
+
+        <FormField
+          label="Feedback"
+          input={
+            <TextInput.Multiline
+              name='feedback'
+              rows={5}
+              style={{ resize: 'none', height: 'auto' }}
+              onChange={this.changeField}
+              placeholder="Do you have any feedback to provide the contributor?"
+              wide
+            />
+          }
+        />
+
+        <ReviewRow>
+          <ReviewButton emphasis="negative" onClick={this.onReviewApplicationReject}>Reject</ReviewButton>
+          <ReviewButton emphasis="positive" onClick={this.onReviewApplicationAccept}>Accept</ReviewButton>
+        </ReviewRow>
+
+        <AlertArea>
+          <RatingAlert alert={this.state.ratingAlert} />
+        </AlertArea>
+        
+      </div>
+    )
+  }
+
+}
+
+function CommentsCheck(props) {
+  if (!props.comments) return null
+  else{
+    return <FieldTitle>Additional Comments</FieldTitle>
+  }
 }
 
+function RatingAlert(props) {
+  return props.alert ? <Info.Alert>Please Select A Quality Rating</Info.Alert> : null
+}
+
+const IssueTitle = styled(Text)`
+  color: #717171;
+  font-size: 17px;
+  font-weight: 300;
+  line-height: 38px;
+`
+const IssueLinkRow = styled.div`
+  height: 31px;
+  display: flex;
+  align-items: center;
+  margin-bottom: 10px;
+`
+const SubmissionDetails = styled.div`
+  border: 1px solid #DAEAEF;
+  background-color: #F3F9FB;
```

Comment:
Should we use theme values here? or maybe these colors are not present on the aragon ui theme